contracts/libraries/OracleLibrary.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+
+import '@uniswap/v3-core/contracts/libraries/FullMath.sol';
+import '@uniswap/v3-core/contracts/libraries/TickMath.sol';
+import '@uniswap/v3-core/contracts/interfaces/pool/IUniswapV3PoolDerivedState.sol';
+import '@uniswap/v3-core/contracts/libraries/LowGasSafeMath.sol';
+import 'hardhat/console.sol';
+
+library OracleLibrary {
+    function consult(
+        uint32 time,
+        address poolAddress,
+        uint32 startBlockTimestamp,
+        uint32 endBlockTimestamp
+    ) public view returns (uint256 ratioX128) {
```

Comment:
just make this method internal and you don't have to deploy the library separately because the bytecode will get inlined in the caller contract